participants/1pan.md

```diff
@@ -0,0 +1,69 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Name:1pan
+Slack:longkengkeng
+
+MemberId:U01BZR3FNPJ
+
+## (1) Brief Project Description
+
+*We wanted to create a disk application on the Filecoin network so that our user data would not be lost. For the foreseeable future, we hope that our application data in the Filecoin gateway will be easily accessible to our users.*
+
+## (2) Link to Project UI
+
+*demo: http://ipfs.forke.cn/*
+
+You can see UI in github:
+
+https://github.com/zlongxiao/1pan.git
+
+## (3) What does your application/UI do?
+
+*This is a traditional disk project, and we really want to see it become a business application on the Filecoin network.The project has login, registration, upload, share, download, reward and other functions.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Various*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will not collect any data, and we will slice the data and store it in the Filecoin network. In the IPFS network, there is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by our staff.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*We now have almost 500G of data, and we plan to upload some of the cold data to the IPFS network. During the use, if the access speed can reach our expectation, we will upload all the data uploaded by users to the IPFS network.*
+
+## (7) How are you structuring the data?
+
+*We cut the data into pieces and store it on the Filecoin network.*
+
+## (8) What pre-processing are you doing before ingesting the data?
+
+*We will use the API provided by you to upload the files to the Filecoin network, and we hope you can help us if we encounter problems in the development process.*
+
+## (9)  What tech stack will you use to store the data?
+
+*Powergate,we will use api to do this.*
+
+## (10) How will you retrieve the data?
+
+*The storage and reading of data depends entirely on our user behavior.*
+
+## (11) Who is the intended user for your application/UI?
+
+*Individual users and enterprise users.*
+
+## (12) Do you have any users already or plans to acquire users soon?
+
+*Yes,this project is running and our users are growing.*
+
+## (13) What challenges do you anticipate with this project?
+
+*The access speed of Filecoin network is the biggest challenge we are facing.*
```

Comment:
Thank you for surfacing this! Is this data retrieval speed that you're referring to? If that's the case, hopefully Powergate should help since it's also storing data in IPFS hotter storage.

Reply:
Yes,I may use Powergate for this application ,thanks for your patience.